annotator-core/src/test/java/edu/ucr/cs/riple/core/tools/CoreTestHelper.java

```diff
@@ -479,7 +479,13 @@ private DEFAULT_PREDICATE(Config config) {
 
     @Override
     public boolean test(TReport expected, Report found) {
-      return expected.root.change.getLocation().equals(found.root.change.getLocation())
+      return expected
+              .root
+              .changes
+              .iterator()
+              .next()
```

Comment:
Should we also assert that the size of `changes` is equal to 1?

Reply:
Actually based on your earlier comment [comment](https://github.com/ucr-riple/NullAwayAnnotator/pull/242#discussion_r1788364974) mentioning that instead of working with set of locations, we can simply use the fix instance, I updated this method to just check the equality of fixes directly rather than the locations.